providers/implementations/ciphers/cipher_aes_siv.c

```diff
@@ -248,7 +250,6 @@ static OSSL_FUNC_cipher_update_fn lc##_stream_update;                          \
 static OSSL_FUNC_cipher_final_fn lc##_stream_final;                            \
 static OSSL_FUNC_cipher_cipher_fn lc##_cipher;                                 \
 static OSSL_FUNC_cipher_get_params_fn alg##_##kbits##_##lc##_get_params;       \
-static OSSL_FUNC_cipher_gettable_params_fn alg##_##lc##_gettable_ctx_params;   \
```

Comment:
? why is this removed?

Reply:
It's a mistake.  It's a `gettable_params` callback (on the EVP_CIPHER) returning the `gettable_ctx_params` parameters (for the EVP_CIPHER_CTX`).

A proper version of this function wasn't defined or used anywhere.